.deploy/dev-gcp.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: familie-dokument
+  namespace: teamfamilie
+  labels:
+    team: teamfamilie
+
+spec:
+  image: {{ image }}
+  team: teamfamilie
+  port: 8082
+  liveness:
+    path: /familie/alene-med-barn/mellomlagring/internal/status/isAlive
+    initialDelay: 30
+    timeout: 5
+    periodSeconds: 10     # How often (in seconds) to perform the probe. Default to 10 seconds
+    failureThreshold: 10 # when a Pod starts and the probe fails,
+  readiness:
+    path: /familie/alene-med-barn/mellomlagring/internal/status/isAlive
+    initialDelay: 30
+    timeout: 5
+    periodSeconds: 10
+    failureThreshold: 10
+  prometheus:
+    enabled: true
+    path: /familie/alene-med-barn/mellomlagring/internal/prometheus
+  vault:
+    enabled: true
+  replicas:
+    min: 2
+    max: 4
+    cpuThresholdPercentage: 80
+  resources:
+    limits:
+      memory: "1024Mi"
+      cpu: "1000m"
+    requests:
+      memory: "512Mi"
+      cpu: "500m"
+  ingresses: # Optional. List of ingress URLs that will route HTTP traffic to the application.
+    - https://familie-dokument.dev-gcp.nais.io
+  accessPolicy:
+    outbound:
+      external:
+        - host: familie-ba-sak.nais.preprod.local
```

Comment:
Shouldn't ba-sak be an inbound accessPolicy?
https://doc.nais.io/nais-application/access-policy

We need ef-soknad-api here as well

Reply:
Because ba-sak is not inside GCP now, it has to be outbound policy (I am also confused with it, and disagree):

> Outbound rules
> Inbound rules specifies what other applications your application receives traffic from. spec.accessPolicy.outbound.rules specifies which applications in the same cluster to open for. To open for external applications, use the field spec.accessPolicy.outbound.external.

Reply:
Ok, seems to be correct! Seems like we do not need to define familie-ba-sak/ef-soknad-api, they use the ingress url which is enough
https://nav-it.slack.com/archives/GREQVPXHS/p1591862129004800?thread_ts=1591861049.003600&cid=GREQVPXHS

Reply:
Ok, I'll try to remove them